src/components/views/elements/ImageView.js

```diff
@@ -72,11 +78,15 @@ module.exports = React.createClass({
         if(this.props.name) {
             name = this.props.name;
         } else if(this.props.mxEvent) {
-            name = props.mxEvent.getContent().body;
+            name = this.props.mxEvent.getContent().body;
```

Comment:
I feel like this component has no business messing with `mxEvent`, and the poking in `mxEvent` should be left to the parent. I'd understand if you didn't fancy messing with this right now, but let's not propagate the evil. I'd rather see this line removed than fixed.